src/WMStream.ts

```diff
@@ -0,0 +1,29 @@
+const util = require('util');
+const stream = require('stream');
+const Writable = stream.Writable;
+
+export default class WMStream extends Writable 
+{
+  _write: any;
+  public memStore: any = {};
```

Comment:
Avoid using abbreviation, also strange that linter hasn't complained about lack of documentation. Make sure you run linter.

Reply:
Abbreviation is still there.

Reply:
i think i removed it, this github change list view isn't great.